packages/compass-e2e-tests/tests/collection-aggregations-tab.test.ts

```diff
@@ -234,19 +234,37 @@ describe('Collection aggregations tab', function () {
       const text = await textElement.getText();
       return text === '(Sample of 100 documents)';
     });
+  });
 
-    // save as a view
-    // TODO: This is currently broken, so will have to test at a later stage
-    /*
-    //#save-pipeline-actions
-    //a=Create a view'
-    '[trackingid="create_view_modal"]'
-    '#create-view-name'
-    '[trackingid="create_view_modal"] [role=dialog] > div:nth-child(2) button:first-child'
-    */
+  it('supports saving pipeline as a view', async function () {
```

Comment:
I was going to just stick it at the end of the previous test since we already have an aggregation built there and then that test acts as one "full flow", but I suppose it doesn't matter.

Reply:
I think I got a bit lost in the code there, I now see that the comment was inside another test, I'll move it